### ISSUE TITLE
[Bug]: Duplicates of Every Song

### ISSUE THREAD

**Issue by jmillerv** [navidrome/navidrome#2508]
### I confirm that:

- [X] I have searched the existing [open AND closed issues](https://github.com/navidrome/navidrome/issues?q=is%3Aissue) to see if an issue already exists for the bug I've encountered
- [x] I'm using the latest version (your issue may have been fixed already)

### Version

0.49.3

### Current Behavior

Each song in my files is included in the UI twice. I double-checked the file system to ensure that songs weren't somehow accidentally duplicated within the folders.  

### Expected Behavior

Each song only appears once. 

### Steps To Reproduce

1. login to the webui 
2. Perform a full scan 
3. Navigate to any given album and noticed that each track is duplicated. 

### Environment

```markdown
- OS:Unraid
- Browser: Firefox Developer Edition/ Firefox
- Client: Navidrome GUI, Symfonium
```

### How Navidrome is installed?

Package

### Configuration

```toml
I'm using the Unraid docker app.
```

### Relevant log output

```shell
When parsing the logs nothing stuck out as obvious.
```

### Anything else?

The songs in the Navidrome GUI
![image](https://github.com/navidrome/navidrome/assets/50455719/9dedaee3-b407-4002-96a0-8442c21b7d62)

The songs in the directory 
![image](https://github.com/navidrome/navidrome/assets/50455719/ac8b5e45-6925-4a1e-b5e7-947abf6ed371)

### Code of Conduct

- [X] I agree to follow Navidrome's Code of Conduct

**Comment by jmillerv** [comment]
Another weird thing. I stopped the container, deleted the appdata and restarted it and had the same issue.

**Comment by jmillerv** [comment]
Is it possible this is a tagging issue? I've noticed that albums from Bandcamp and one from the Internet Archive display properly. The rest are from CDs I archived. This hasn't been the case until recently though and I cannot figure out why it is doing it.

**Comment by jmillerv** [comment]
I had some time to investigate this evening my hunch about tags seems to be a direction worth pursuing. 

Only appears once. 
![image](https://github.com/navidrome/navidrome/assets/50455719/5ae6cc22-7e59-4aa4-8f1d-edede88558d9)

![image](https://github.com/navidrome/navidrome/assets/50455719/b22c022a-38e2-4a21-8901-00f66bddc36d)


Appears twice. 
![image](https://github.com/navidrome/navidrome/assets/50455719/155cc6c7-8477-4fce-95ce-4b89bffe0dbd)

![image](https://github.com/navidrome/navidrome/assets/50455719/1d5a5bcc-79d5-4c33-bd2c-12316605dec2)

**Comment by jmillerv** [comment]
I went to an album with both ID3v1 & ID3v2.3 tags and removed the ID3v1. I now have a better but weirder problem. The better part is that instead of duplicating the entire tracks in order so instead of Track 1, Track 1, Track 2, etc 

It has the album as two discs with the entire song. Which is a better listening experience (end of one track flows into the expected next track). It's still not correct.

**Comment by metalheim** [comment]
Are your entirely sure that there is no 2nd copy of these files in your library?
I have a big library and never have seen this happen. 

What you can do to troubleshoot is:
- go into the "Players" Settings
- choose your currently used web browser (sort by "last seen" - the newest one)
- enable option "report real paths"

Then in the album, when you right-click a song in navidrome, it will show you where the file is located.
- compare the path one song to it's duplicate

If the file paths are exactly the the same, maybe its worth "touch" ing all files, so navidrome registers them as changed. 
It might be that there is a weird thing with unraid -> docker -> volume mounts where the file date properties arent properly updated or something.

**Comment by jmillerv** [comment]
I think you're onto something. I still don't think there are duplicates in the file system, but one duplicate song had the path I expected, and the other had no path at all.

**Comment by jmillerv** [comment]
Okay, yeah, I think this might be an unraid shares issue. Going through more songs I see a consistent pattern. Some container setting is cached somewhere. The mount should be `/music` and I think on a previous set-up it ended up being `/music/Music` and I updated it just `/music` to avoid redundancy. 

I'll go ahead and close this and figure out what's going on with unraid shares and docker caching. Thanks for the help!

**Comment by jmillerv** [comment]
Just going to update this for anyone who might stumble upon and come through. There were no duplicates in the file system, but when I redid the mount, the old path persisted. 

The share I mounted to `/music` somehow cached an older version and there as a whole `Music` folder which included a lot of the old stuff which caused the duplicate. 

I fixed this by logging into the container and checking the file list in mount `/music` and then running an `rm -rf /music/Music`
